packages/frontend/src/utils/colorUtils.ts

```diff
@@ -14,41 +14,49 @@ export const readableColor = (backgroundColor: string) => {
     return onWhite > onBlack ? 'white' : 'black';
 };
 
-const getColorRange = (
-    colorConfig: ConditionalFormattingColorRange,
-): string[] | undefined => {
+const getColorRange = (colorConfig: ConditionalFormattingColorRange) => {
     if (
         !isHexCodeColor(colorConfig.start) ||
         !isHexCodeColor(colorConfig.end)
     ) {
         return undefined;
     }
 
-    const colors = Color.steps(
+    return Color.range(
         new Color(colorConfig.start),
         new Color(colorConfig.end),
         {
-            steps: colorConfig.steps,
             space: 'srgb',
         },
     );
-
-    return colors.map((c) => new Color(c).toString({ format: 'hex' }));
 };
 
 export const getColorFromRange = (
     value: number,
     colorRange: ConditionalFormattingColorRange,
     minMaxRange: ConditionalFormattingMinMax,
 ): string | undefined => {
-    const colors = getColorRange(colorRange);
-    if (!colors) return undefined;
+    const interpolateColor = getColorRange(colorRange);
+
+    if (!interpolateColor) return undefined;
 
     const min = minMaxRange.min;
-    const inclusiveMax = minMaxRange.max + 1;
```

Comment:
`+ 1` was breaking it for small numbers. Not sure what was the reasoning behind it 🤔